backend/config/config_template.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
```

Comment:
Integrate the `config_template` properly in the config module (backend/capellacollab/config) is the right module.

Reply:
I wasn't sure if you mean actually putting the config_template in backend/capellacollab/config, but that gets messy when trying to generate the config. I think ideally we'd tell users to run `python -m capellacollab.config.config_template`, but the config module needs a config to already exist for  __init__.py so it will just fail. The workarounds I thought of were to tell users to run the script directly instead of as a module (`python backend/capellacollab/config/config_template.py` seems less elegant), checking what file was calling the module and skipping the config load if it is config_template (seems hacky), or instead of failing to launch the app when no config exists to just pass a default AppConfig without saving it to file and leaving config tweaking to advanced users entirely. Any thoughts?

Reply:
`python -m capellacollab.config.config_template` looks a bit weird as `config` is mentioned twice and the intention isn't clear. What about `python -m capellacollab.config.generate`? Models always belong to the `models.py` in the specific module. 

> config module needs a config to already exist for init.py so it will just fail when someone with a fresh repo copy tries it

You can move the config loading to the `__main__.py`

> instead of failing to launch the app when no config exists to just pass a default AppConfig without saving it to file and leaving config tweaking to advanced users entirely.

Thought about that too, but then the user doesn't even know that a configuration exists and how to modify it (if it's a user action to generate it, it's clear) and if the config is placed at a wrong location, it would start with the defaults and the user might not notice it directly, is also hard to debug.

Reply:
Excellent points, I'll follow this and then I think(?) I'll be set to submit for review, though I saw some weird behavior in diff.py I might need to check first (it wasn't catching new or deleted keys for some reason).